tools/build/options/WITH_MALLOC_REVOCATION_SYSTEM

```diff
@@ -0,0 +1,2 @@
+Enable the malloc revocation system (mrs) wrapper for allocators that
```

Comment:
Isn't the S for Shim?

Reply:
Hmm, I'm sure I found system somewhere, but now I'm only seeing shim. Will update appropriately.